models/git/commit_status.go

```diff
@@ -197,6 +208,21 @@ func (status *CommitStatus) LocaleString(lang translation.Locale) string {
 	return lang.TrString("repo.commitstatus." + status.State.String())
 }
 
+// HideActionsURL set `TargetURL` to an empty string if the status comes from Gitea Actions
+func (status *CommitStatus) HideActionsURL(ctx context.Context) {
+	if status.Repo == nil {
+		if err := status.loadRepository(ctx); err != nil {
+			log.Error("loadRepository: %v", err)
+			return
+		}
+	}
+
+	prefix := fmt.Sprintf("%s/actions", status.Repo.Link())
+	if strings.HasPrefix(status.TargetURL, prefix) {
```

Comment:
Is `TargetURL` a relative path or a URL?

Reply:
It depends on the source of the commit status. 
- If the commit status comes from Gitea Actions, the `TargetURL` will be a relative path like `/{owner}/{repo}/actions/runs/1/jobs/2`. 
- If the commit status comes from an external service like Drone, the `TargetURL` will be a URL.